README.md

```diff
@@ -59,7 +59,29 @@ man f3d # Linux only
 
 F3D is developed by a team of passionate devs. Please use F3D, star it on github or even become a [sponsor](https://github.com/sponsors/f3d-app) to support us!
 
-## Contributing
+# Goals/Non-goals
+
+We believe that clearly stating the goals and non-goals of F3D would help potential users and developers figure out if F3D is the right tool for them.
+
+## Goals
+
+- Support as many 3D file formats as possible
+- Support many types of visualization
+- Support all industries dealing with 3D datasets
```

Comment:
I know what you mean but "industries" sounds a bit strong/specific; could it be reworded with something more like "use-case"/"scenario"/"workflow"/...?